products/sentinel_one.py

```diff
@@ -205,6 +205,11 @@ def build_query(self, filters: dict) -> Tuple[str, datetime, datetime]:
                     query_base += ' AND '
 
                 query_base += f' UserName containscis "{value}"'
+            elif key == 'process_name':
```

Comment:
To support this new filter, we'll need to add that option to `surveyor.py` and the other three EDR products

Reply:
this change was reverted as it was unneeded.